irc/handlers.go

```diff
@@ -1286,16 +1287,16 @@ func kickHandler(server *Server, client *Client, msg ircmsg.Message, rb *Respons
 		channel string
 		nick    string
 	}
-	kicks := make([]kickCmd, 0, len(channels))
-	for index, channel := range channels {
+	kicks := make([]kickCmd, 0, len(users))
+	channel := channels[0]
+	for index, user := range users {
```

Comment:
From the code it looks like previously you could `KICK #channel1,#channel2 user`. Looks like this will now only kick them from #channel1?

Reply:
Indeed, I didn't consider this. I don't think that's a great feature to have (it can be confusing), but I just restored it.